application/controllers/Login.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+class Login extends FrameWork {
+
+  public function authenticate() {
+    if (isset($_POST['mailid'])) {
+      if ($this->model('Database')){
+        $connect = new Database();
+        if ($connect->verifyEmail($_POST['mailid'])) {
```

Comment:
Please check out this comment ,https://github.com/Eshakundu01/Social_site_MVC/pull/1#discussion_r1141732452